forms.rst

```diff
@@ -696,10 +696,9 @@ Learn more
     :glob:
 
     form/*
-
-* :doc:`/controller/upload_file`
-* :doc:`/reference/forms/types`
-* :doc:`/http_cache/form_csrf_caching`
+    /controller/upload_file
```

Comment:
Is that slash at the beginning needed? since `form/*` doesn't have it?

Reply:
Good catch! I've added the leading slash to forms.

Also, it looks like all formats are supported. Most `toctree` add a leading `/`, some don't. Most `toctree` don't add `.rst` suffix. Some do.